Gemfile

```diff
@@ -29,7 +29,7 @@ gem "ruport",                         "=1.7.0",                       :git => "g
 
 # Vendored but not required
 gem "net-ldap",                       "~>0.7.0",   :require => false
-gem "rubyrep",                        "=1.2.0",    :require => false, :git => "git://github.com/ManageIQ/rubyrep.git", :tag => "v1.2.0-7"
+gem "rubyrep",                        "=1.2.0",    :require => false, :git => "git://github.com/matthewd/rubyrep.git", :branch => "rails5"
```

Comment:
Can we get the rubyrep changes into manageiq repo?
Are they 2.4 friendly?
Maybe have a rails 5 branch over there?